renderers/src/main/java/com/pedrogomez/renderers/RVListRendererAdapter.java

```diff
@@ -0,0 +1,215 @@
+package com.pedrogomez.renderers;
+
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v7.recyclerview.extensions.AsyncDifferConfig;
+import android.support.v7.recyclerview.extensions.ListAdapter;
+import android.support.v7.util.DiffUtil;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import com.pedrogomez.renderers.exception.NullRendererBuiltException;
+
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * ListAdapter extension created to work with RendererBuilders and Renderer instances. Other
+ * adapters have to use this one or RVRendererAdapter to show information into RecyclerView widgets.
+ * <p>
+ * This class is used to avoid the library users declare a new renderer each time they have to show
+ * information into a RecyclerView. The main difference between RVRendererAdapter and RVListRendererAdapter
+ * is that RVListRendererAdapter performs the adapter elements' diffing calculation in a background thread.
+ * <p>
+ * RVListRendererAdapter has to be constructed with a RendererBuilder to provide Renderer
+ * to RvListRendererAdapter, one of DiffUtil.ItemCallback or AsyncDifferConfig to provide the
+ * diffing configuration and one AdapteeCollection to provide the elements to render.
+ *
+ * @author Víctor Julián García Granado.
+ */
+public class RVListRendererAdapter<T> extends ListAdapter<T, RendererViewHolder> {
```

Comment:
It's basically the same code as `RVRendererAdapter` with changes on constructors and diffing calculation.